### DISCUSSION TITLE
My API URL is returning null

### DISCUSSION THREAD

**Discussion by shamxeed** [deta/cloud-docs#426] [Help]
I don't know what might be the issues, I have successfully deployed my node app but after testing out the URL it is returning null.

**Comment by shamxeed** [comment]
After enabling logging, I realized that the script commands in my package.json file were not executed. And I have not seen a document where I can archive that. Any help will be much appreciated!!

**Reply by Abban-Fahim** [reply]
node js apps work by using the object u export from your entry point (index.js). If you are using express, just export the instance of express (app) and deta will run it for you! 

Check their code examples and supported libraries here: https://docs.deta.sh/docs/micros/deploy#example-code-1

**Reply by shamxeed** [reply]
I'm getting `Micro timed out.` now.
